### PR TITLE
Create navbar automation

### PR DESCRIPTION
**Pull request by MichalWegielski**
### GOAL

Navbar automation is ready.

### PROOF OF IMPLEMENTATION 
prooff

### TRELLO 

https://trello.com/c/5LvQ6d8l/560-navbar-automation

### CODE REVIEW

**Comment by MichalWegielski** [comment]
There is strange situation where Js code cant pass eslint test instead of I repair all errors. It looks like code automaticly add blank space before and after "{" brackets on github platform.

File: assets/js/admin/topbar.js
Comment (KiviGibson) [inline assets/js/admin/topbar.js:1]: Check errors in test before requesting to merge. Use quality command. You can get the command after you source the .profile file.

File: assets/js/website/navbar.js
Comment (KiviGibson) [inline assets/js/website/navbar.js:1]: same as above

**Review by KiviGibson — CHANGES_REQUESTED** [review]
Use quality command to check for errors in your code before merging.

**Review by KiviGibson — APPROVED** [review]

**Pull request merged**